src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion_k_diffusion.py

```diff
@@ -456,7 +458,10 @@ def __call__(
             callback_steps (`int`, *optional*, defaults to 1):
                 The frequency at which the `callback` function will be called. If not specified, the callback will be
                 called at every step.
-
+            use_karras_sigmas (`bool`, *optional*, defaults to `False`):
+                Use karras sigmas. For example, specifying `sample_dpmpp_2m` to `set_scheduler` will be equivalent to
+                `DPM++2M` in stable-diffusion-webui. On top of that, setting this option to True will make it `DPM++2M
+                Karras`.
```

Comment:
This is how docstrings should be I think. 

Could you maybe just add a link to the paper that introduced Karras sigmas?